### ISSUE TITLE
Deploy to Heroku

### ISSUE THREAD

**Issue by Qoyyuum** [notjacobjun/Solitude-admin#19]
Set it up to auto deploy to Heroku

**Comment by Qoyyuum** [comment]
> Set it up to auto deploy to Heroku

@notjacobjun please assign me this issue. Thanks

**Comment by Qoyyuum** [comment]
Blocked by #20

**Comment by Qoyyuum** [comment]
@notjacobjun @avinashupadhya99  Is this working on your heroku instance? What error or issue are you facing with this now?

**Comment by notjacobjun** [comment]
I dont believe that it is working right on heroku however I will have to double check
